contracts/tests/fixtures_uraiden.py

```diff
@@ -73,52 +77,57 @@ def uraiden_instance(owner, uraiden_contract):
     return uraiden_instance
 
 
-@pytest.fixture(params=['20', '223'])
-def get_channel(web3, request, owner, get_accounts, event_handler, print_gas, txn_gas, get_block):
-    def get(uraiden_instance, token_instance, deposit, sender=None, receiver=None):
-        contract_type = request.param
-        ev_handler = event_handler(uraiden_instance)
+@pytest.fixture
+def get_channel(web3, channel_params, owner, get_accounts, uraiden_instance, token_instance, event_handler, print_gas, txn_gas, get_block):
```

Comment:
style: Too many args. Split in multiple lines with each arg in its own line (assuming you follow the raiden styling guide)

Reply:
Yes, only ran flake8 on test_channel_close.py at the time... But I refactored this part in https://github.com/raiden-network/microraiden/pull/310 . I would leave it like this for now, to have less merging conflicts.